brew/stop-uninstall-wipe.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+set -u # also -e?
+
+eval $(brew/vars.sh)
+
+echo
+echo stopping...
+sudo brew services stop tailscale
+sudo brew services list | grep tailscale
+ps -ef | grep tailscaled | grep -v "grep tailscaled"
+
+echo
+echo uninstalling...
+brew uninstall --force tailscale
+sudo brew services list | grep tailscale
+
+echo
+echo deleting...
+sudo rm -rf $TS_CELLAR/$TS_VER
+sudo rm -f /usr/local/var/lib/tailscale/*
```

Comment:
This is not portable to M1 Macs either. All of the `/usr/local` paths should be in `/opt/homebrew` on M1 Macs. Maybe you should have a `$HOMEBREW_ROOT` variable?